### PR TITLE
update with remaining JS highlights

### PR DESCRIPTION
**Pull request by KarishmaGhiya**
- All GAs or first beta of JS package has been included in the release highlights.

### CODE REVIEW

**Review by ellismg — APPROVED** [review]
EG changes look good, nothing else to call out.

**Review by sarangan12 — APPROVED** [review]

File: releases/2021-06/index.md [hunk 1]
```diff
@@ -18,7 +18,13 @@ You can find links to packages, code, and docs on our [Azure SDK Releases page](
...
+- Models Repository service has released a first beta for JS package @azure/iot-modelsrepository. This package contains the `ModelsRepositoryClient` to talk to the Azure Models Repository service, with initial support for getting models and helper functions for working with DTMIs.
 - Cognitive Search SDK for JS has released a new GA version 11.2.0.
+- Eventgrid SDK for JS has released a new GA version 4.3.0.
+- Storage has released new GA versions for @azure/storage-file-share, @azure/storage-blob, @azure/storage-file-datalake, @azure/storage-queue SDKs with Azure Storage Service API version 2020-08-04.
+- Tables SDK for JS has released a new GA version 12.0.0. It has fixes for transaction issues when there are multiple transactions and date serializations, added support for generating SAS tokens.
+- Service Bus SDK for JS has released a new GA version 7.2.0. It enables encoding the body of a message, which allows you to take advantage of native AMQP serialization for supported primitives or sequences.It improves cancellation support when sending messages or initializing a connection to the service. This version has a few bug fixes too.
```
Comment (richardpark-msft) [inline releases/2021-06/index.md:27]:
```suggestion
- Service Bus SDK for JS has released a new GA version: 7.2.0. It enables encoding the body of a message using native AMQP serialization for supported primitives or sequences. In addition, cancellation support has been improved when sending messages or initializing a connection to the service.
```

Maybe?

**Review by richardpark-msft — APPROVED** [review]
Slight update suggestion for the Service Bus portion.

File: releases/2021-06/index.md [hunk 2]
```diff
@@ -18,7 +18,13 @@ You can find links to packages, code, and docs on our [Azure SDK Releases page](
   - Ingest farm operations data covering tilling, planting, harvesting and application of farm inputs.
 - Form Recognizer SDK for JS released a new GA version 3.1.0. This General Availability (GA) release marks the stability of the changes introduced in package versions 3.1.0-beta.1 through 3.1.0-beta.3.
 - Communication Relay has released a first beta for JS package @azure/communication-network-traversal. Azure Communication Network Traversal is managing tokens for Azure Communication Services.
+- Monitor Query has released a first beta for JS package, which allows you to query Log Analytics Workspaces for logs and metrics.
```
Comment (richardpark-msft) [inline releases/2021-06/index.md:21]:
```suggestion
- Monitor Query (`@azure/monitor-query`) has released 1.0.0-beta.1, which allows you to query Log Analytics Workspaces for logs and metrics.
```
```suggestion
- Monitor Query has released a first beta for JS package, which allows you to query Log Analytics Workspaces for logs and metrics.
```
```suggestion
- Monitor Query (`@azure/monitor-query`) has released a first beta for JS package, which allows you to query Log Analytics Workspaces for logs and metrics.
```

**Comment by KarishmaGhiya** [comment]
@witemple-msft  If you have any updates/ additions for release highlights for `Form-recognizer` highlights that might be useful for customers to see then please ping me I can update in a follow up PR. 

@joheredi   If you have any updates/ additions for release highlights for `Tables` or `Farm-beats` highlights that might be useful for customers to see then please ping me I can update in a follow up PR.

File: releases/2021-06/index.md [hunk 3]
```diff
@@ -18,7 +18,13 @@ You can find links to packages, code, and docs on our [Azure SDK Releases page](
...
 - Communication Relay has released a first beta for JS package @azure/communication-network-traversal. Azure Communication Network Traversal is managing tokens for Azure Communication Services.
+- Monitor Query (`@azure/monitor-query`) has released 1.0.0-beta.1, which allows you to query Log Analytics Workspaces for logs and metrics.
+- Models Repository service has released a first beta for JS package @azure/iot-modelsrepository. This package contains the `ModelsRepositoryClient` to talk to the Azure Models Repository service, with initial support for getting models and helper functions for working with DTMIs.
 - Cognitive Search SDK for JS has released a new GA version 11.2.0.
+- Eventgrid SDK for JS has released a new GA version 4.3.0.
+- Storage has released new GA versions for @azure/storage-file-share, @azure/storage-blob, @azure/storage-file-datalake, @azure/storage-queue SDKs with Azure Storage Service API version 2020-08-04.
+- Tables SDK for JS has released a new GA version 12.0.0. It has fixes for transaction issues when there are multiple transactions and date serializations, added support for generating SAS tokens.
```
Comment (KarishmaGhiya) [inline releases/2021-06/index.md:26]:
```suggestion
- Tables SDK for JS (`@azure/data-tables`) has released a new GA version: 12.0.0. It has fixes for transaction issues when there are multiple transactions and date serializations, added support for generating SAS tokens.
```

File: releases/2021-06/index.md [hunk 4]
```diff
@@ -18,7 +18,13 @@ You can find links to packages, code, and docs on our [Azure SDK Releases page](
...
 - Form Recognizer SDK for JS released a new GA version 3.1.0. This General Availability (GA) release marks the stability of the changes introduced in package versions 3.1.0-beta.1 through 3.1.0-beta.3.
 - Communication Relay has released a first beta for JS package @azure/communication-network-traversal. Azure Communication Network Traversal is managing tokens for Azure Communication Services.
+- Monitor Query (`@azure/monitor-query`) has released 1.0.0-beta.1, which allows you to query Log Analytics Workspaces for logs and metrics.
+- Models Repository service has released a first beta for JS package @azure/iot-modelsrepository. This package contains the `ModelsRepositoryClient` to talk to the Azure Models Repository service, with initial support for getting models and helper functions for working with DTMIs.
 - Cognitive Search SDK for JS has released a new GA version 11.2.0.
+- Eventgrid SDK for JS has released a new GA version 4.3.0.
+- Storage has released new GA versions for @azure/storage-file-share, @azure/storage-blob, @azure/storage-file-datalake, @azure/storage-queue SDKs with Azure Storage Service API version 2020-08-04.
```
Comment (KarishmaGhiya) [inline releases/2021-06/index.md:25]:
```suggestion
- Storage has released new GA versions for `@azure/storage-file-share`, `@azure/storage-blob`, `@azure/storage-file-datalake`, `@azure/storage-queue` SDKs with Azure Storage Service API version 2020-08-04.
```

File: releases/2021-06/index.md [hunk 5]
```diff
@@ -18,7 +18,13 @@ You can find links to packages, code, and docs on our [Azure SDK Releases page](
   - Ingest farm operations data covering tilling, planting, harvesting and application of farm inputs.
 - Form Recognizer SDK for JS released a new GA version 3.1.0. This General Availability (GA) release marks the stability of the changes introduced in package versions 3.1.0-beta.1 through 3.1.0-beta.3.
 - Communication Relay has released a first beta for JS package @azure/communication-network-traversal. Azure Communication Network Traversal is managing tokens for Azure Communication Services.
+- Monitor Query (`@azure/monitor-query`) has released 1.0.0-beta.1, which allows you to query Log Analytics Workspaces for logs and metrics.
+- Models Repository service has released a first beta for JS package @azure/iot-modelsrepository. This package contains the `ModelsRepositoryClient` to talk to the Azure Models Repository service, with initial support for getting models and helper functions for working with DTMIs.
 - Cognitive Search SDK for JS has released a new GA version 11.2.0.
+- Eventgrid SDK for JS has released a new GA version 4.3.0.
```
Comment (KarishmaGhiya) [inline releases/2021-06/index.md:24]:
```suggestion
- Eventgrid SDK for JS (`@azure/eventgrid`) has released a new GA version: 4.3.0.
```

File: releases/2021-06/index.md [hunk 6]
```diff
@@ -18,7 +18,13 @@ You can find links to packages, code, and docs on our [Azure SDK Releases page](
   - Ingest farm operations data covering tilling, planting, harvesting and application of farm inputs.
 - Form Recognizer SDK for JS released a new GA version 3.1.0. This General Availability (GA) release marks the stability of the changes introduced in package versions 3.1.0-beta.1 through 3.1.0-beta.3.
 - Communication Relay has released a first beta for JS package @azure/communication-network-traversal. Azure Communication Network Traversal is managing tokens for Azure Communication Services.
+- Monitor Query (`@azure/monitor-query`) has released 1.0.0-beta.1, which allows you to query Log Analytics Workspaces for logs and metrics.
+- Models Repository service has released a first beta for JS package @azure/iot-modelsrepository. This package contains the `ModelsRepositoryClient` to talk to the Azure Models Repository service, with initial support for getting models and helper functions for working with DTMIs.
 - Cognitive Search SDK for JS has released a new GA version 11.2.0.
```
Comment (KarishmaGhiya) [inline releases/2021-06/index.md:23]:
```suggestion
- Cognitive Search SDK for JS (`@azure/search-documents`) has released a new GA version: 11.2.0.
```

File: releases/2021-06/index.md [hunk 7]
```diff
@@ -18,7 +18,13 @@ You can find links to packages, code, and docs on our [Azure SDK Releases page](
   - Ingest farm operations data covering tilling, planting, harvesting and application of farm inputs.
 - Form Recognizer SDK for JS released a new GA version 3.1.0. This General Availability (GA) release marks the stability of the changes introduced in package versions 3.1.0-beta.1 through 3.1.0-beta.3.
```
Comment (KarishmaGhiya) [inline releases/2021-06/index.md:19]:
```suggestion
- Form Recognizer SDK for JS (`@azure/ai-form-recognizer`) released a new GA version: 3.1.0. This General Availability (GA) release marks the stability of the changes introduced in package versions 3.1.0-beta.1 through 3.1.0-beta.3.
```

**Pull request merged**